compsets/fv3mom6cice5.input

```diff
@@ -61,7 +61,7 @@ build fv3_mom6_cice.exe {
  
     # The NEMSAppBuilder is another embedded bash script in nemscommon.input 
     build=NEMSAppBuilder(NEMS.x="@[target]",modules.nems="@[modules.nems]", 
-                         OPTS="app=coupledFV3_MOM6_CICE",md5sum="@[md5sum]") 
+                         OPTS="app=coupledFV3_MOM6_CICE rebuild",md5sum="@[md5sum]") 
```

Comment:
Why is "rebuild" added here?  In https://github.com/ufs-community/ufs-s2s-model/blob/develop/compsets/nemscommon.input#L34 we have that all builds have to "rebuild"?  This was added in the last commit because when adding a build with WW3, it was required that we rebuild always or we were having issues.

Reply:
Oops. I think this was by coincidence after creating the CCPP versions and then trying to make them look the same. I can revert this change (and make the same change on the CCPP side).

Reply:
It seems that the "rebuild" can then be removed from the other build commands as well, or, given that it is in nemscommon.input?